source/exe/terminate_handler.cc

```diff
@@ -2,14 +2,28 @@
 
 #include <cstdlib>
 
+#include "envoy/common/exception.h"
 #include "source/common/common/logger.h"
 #include "source/server/backtrace.h"
 
+#include "absl/strings/str_format.h"
+
 namespace Envoy {
 
 std::terminate_handler TerminateHandler::logOnTerminate() const {
   return std::set_terminate([]() {
-    ENVOY_LOG(critical, "std::terminate called! (possible uncaught exception, see trace)");
+    auto currentException = std::current_exception();
```

Comment:
s/currentException/current_exception/

Also I'd use an explicit type here ([std::exception_ptr](https://en.cppreference.com/w/cpp/error/exception_ptr)) per style guide; it's not obvious from call-site.

Reply:
done